### PR TITLE
Deploy Libplanet.RocksDBStore to NuGet

### PR DESCRIPTION
**Pull request by earlbread**
This deploys Libplanet.RocksDBStore to NuGet with the same package version as Libplanet.

### CODE REVIEW

File: .github/bin/dist-nuget.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Note that this script is intended to be run by GitHub Actions.
 set -e
 
-project=Libplanet
+projects="Libplanet Libplanet.RocksDBStore"
```
Comment (dahlia) [inline .github/bin/dist-nuget.sh]: How about using Bash's proper array:

```suggestion
projects=( "Libplanet" "Libplanet.RocksDBStore" )
```

File: .github/bin/dist-nuget.sh [hunk 2]
```diff
@@ -30,7 +30,10 @@ if [ "$GITHUB_REPOSITORY" != "planetarium/libplanet" ] || (
 fi
 
 package_version="$(cat obj/package_version.txt)"
-dotnet nuget push \
-  "./$project/bin/$configuration/$project.$package_version.nupkg" \
-  --api-key "$NUGET_API_KEY" \
-  --source https://api.nuget.org/v3/index.json
+
+for project in $projects; do
```
Comment (dahlia) [inline .github/bin/dist-nuget.sh]: Using the proper array:

```suggestion
for project in "${projects[@]}"; do
```

File: .github/bin/dist-pack.sh [hunk 3]
```diff
@@ -3,29 +3,31 @@
 # Note that this script is intended to be run by GitHub Actions.
 set -e
 
-project=Libplanet
+projects="Libplanet Libplanet.RocksDBStore"
```
Comment (dahlia) [inline .github/bin/dist-pack.sh]:
```suggestion
projects=( "Libplanet" "Libplanet.RocksDBStore" )
```

File: .github/bin/dist-pack.sh [hunk 4]
```diff
@@ -3,29 +3,31 @@
...
 configuration=Release
 
 if ! (env | grep '^GITHUB_'); then
   echo "This script is intended to be run by GitHub Actions." > /dev/stderr
   exit 1
 fi
 
-rm -rf "./$project/bin/$configuration/"
+for project in $projects; do
```
Comment (dahlia) [inline .github/bin/dist-pack.sh]:
```suggestion
for project in "${projects[@]}"; do
```

File: .github/bin/dist-nuget.sh (hunk 1 above)
Comment (earlbread) [reply .github/bin/dist-nuget.sh]: We're using POSIX sh which is not supporting arrays. Would it be better to change to bash instead?

Comment (earlbread) [reply .github/bin/dist-nuget.sh]: I added a commit to use bash array. 1f6aa35cbd66b5095b46bc1014b590c8974a5511

**Pull request merged**